### ISSUE TITLE
Accessibility issue in Wizard: The A is focusable but lacks either an href or role attribute.

### ISSUE THREAD

**Issue by markywilson** [SAP/fundamental-ngx#12473]
![image](https://github.com/user-attachments/assets/2fdc42ca-e495-45aa-a116-5b6f4d6e43fb)

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Latest 

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@markywilson can you provide a little more information and whether you can set that in the markup?
